modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -82,79 +81,91 @@ export interface TabsItemProps
   tabIndex?: number;
 }
 
-export const StyledTabItem = styled(Box.as('button'))<StyledType & Pick<TabsItemProps, 'gap'>>(
-  ({theme}) => ({
-    ...type.levels.subtext.large,
-    fontWeight: type.properties.fontWeights.medium,
+const tabItemStencil = createStencil({
+  base: {
+    ...system.type.subtext.large,
+    fontWeight: system.fontWeight.medium,
     border: 'none',
     backgroundColor: 'transparent',
     flex: '0 0 auto',
+    minWidth: system.space.zero,
     alignItems: 'center',
-    padding: `${space.xs} ${space.s}`,
+    padding: `${system.space.x3} ${system.space.x4}`,
     height: 52,
```

Comment:
height prob needs either a calc or `px2rem`

Reply:
px2rem will be more readable probably